artifacts/definitions/Windows/EventLogs/ScheduledTasks.yaml

```diff
@@ -45,7 +45,7 @@ parameters:
   - name: TaskSchedulerEventRegex
     description: Regex of TaskScheduler log event ids.
     type: regex
-    default: ^(140|141)$
```

Comment:
Maybe make it a choice selector? Otherwise the user has to know 140 and 141 specifically

Reply:
I was changing this to all as default "." as some users may be confused thinking the example eids are it.